cfme/storage/volume.py

```diff
@@ -466,6 +466,7 @@ def instance_count(self):
             :py:class:`int` instance count.
         """
         view = navigate_to(self, 'Details')
+        view.browser.refresh()
```

Comment:
Why is this browser refresh necessary?

Reply:
Because in wait_for when it's already navigated in (self, 'Details') it does nothing when it tries to access it again.
I need the page to be refreshed so I can check current instance_count.

Reply:
Yes, that helped.